pages/api/plants/index.js

```diff
@@ -0,0 +1,18 @@
+import dbConnect from "../../../db/connect";
+import Plant from "@/db/models/Plant";
+
+export default async function handler(request, response) {
+  console.log("API Request received:", request.method);
+  await dbConnect();
+
+  if (request.method === "GET") {
+    const plants = await Plant.find();
```

Comment:
```suggestion
    const plants = await Plant.find().sort({ createdAt: -1 });
```
Update the endpoint to return plants sorted by most recently added, ensuring the newest entries appear first, in line with the user story:
> Once submitted, the newly added plant appears at the top of the plant list.